src/Container.js

```diff
@@ -1,30 +1,18 @@
-import React, { PureComponent } from "react";
-import PropTypes from "prop-types";
-import raf from "raf";
+import React, { PureComponent } from 'react';
 
-export default class Container extends PureComponent {
-  static childContextTypes = {
-    subscribe: PropTypes.func,
-    unsubscribe: PropTypes.func,
-    getParent: PropTypes.func
-  };
+export const StickyContext = React.createContext();
 
-  getChildContext() {
-    return {
-      subscribe: this.subscribe,
-      unsubscribe: this.unsubscribe,
-      getParent: this.getParent
-    };
-  }
+export default class Container extends PureComponent {
+  currentNode = React.createRef();
 
   events = [
-    "resize",
```

Comment:
Looks like your prettier settings are overriding what was used previously. I can add a prettierconfig later to prevent this from happening again, but please use Prettier defaults for now.